official-git/run-tests.sh

```diff
@@ -52,47 +54,95 @@ GIT_SKIP_TESTS="$GIT_SKIP_TESTS t1308.2[6-7]" # No support for line ending handl
 GIT_SKIP_TESTS="$GIT_SKIP_TESTS t1503.[5-7] 1503.10" # No support for @{suffix} (looking up based on reflog) in rev-parse
 GIT_SKIP_TESTS="$GIT_SKIP_TESTS t2018.[6-7] t2018.[9] t2018.1[5-8]"
 GIT_SKIP_TESTS="$GIT_SKIP_TESTS t3000.7"
+GIT_SKIP_TESTS="$GIT_SKIP_TESTS t5510.[4-9] t5510.[1-7][0-9]" # Just the basic fetch tests are working for now
 export GIT_SKIP_TESTS
 
-make t0000-basic.sh \
-        t0004-unwritable.sh \
-        t0007-git-var.sh \
-        t0008-ignores.sh \
-        t0010-racy-git.sh \
-        t0062-revision-walking.sh \
-        t0070-fundamental.sh \
-        t0081-line-buffer.sh \
-        t1000-read-tree-m-3way.sh \
-        t1001-read-tree-m-2way.sh \
-        t1002-read-tree-m-u-2way.sh \
-        t1003-read-tree-prefix.sh \
-        t1004-read-tree-m-u-wf.sh \
-        t1005-read-tree-reset.sh \
-        t1008-read-tree-overlay.sh \
-        t1009-read-tree-new-index.sh \
-        t1011-read-tree-sparse-checkout.sh \
-        t1012-read-tree-df.sh \
-        t1014-read-tree-confusing.sh \
-        t1100-commit-tree-options.sh \
-        t1304-default-acl.sh \
-        t1308-config-set.sh \
-        t1403-show-ref.sh \
-        t1500-rev-parse.sh \
-        t2000-checkout-cache-clash.sh \
-        t2001-checkout-cache-clash.sh \
-        t2002-checkout-cache-u.sh \
-        t2006-checkout-index-basic.sh \
-        t2009-checkout-statinfo.sh \
-        t2014-switch.sh \
-        t2018-checkout-branch.sh \
-        t2100-update-cache-badpath.sh \
-        t3000-ls-files-others.sh \
-        t3002-ls-files-dashpath.sh \
-        t3006-ls-files-long.sh \
-        t3020-ls-files-error-unmatch.sh \
-        t3800-mktag.sh \
-        t4113-apply-ending.sh \
-        t4123-apply-shrink.sh \
-        t7062-wtstatus-ignorecase.sh \
-        t7511-status-index.sh
+echo t0000-basic
+./t0000-basic.sh
```

Comment:
Don't these all need to be strung together with `&&` (or each have an `|| exit 1`) in order to be sure that it's not just the last one that makes Travis fail?

Reply:
I think that the set -e at the beginning of the script ensures this.